crates/store/re_video/src/demux/mod.rs

```diff
@@ -60,12 +60,57 @@ pub struct VideoData {
     /// and should be presented in composition-timestamp order.
     pub samples: Vec<Sample>,
 
+    /// Meta information about the samples.
+    pub sample_statistics: SampleStatistics,
+
     /// All the tracks in the mp4; not just the video track.
     ///
     /// Can be nice to show in a UI.
     pub mp4_tracks: BTreeMap<TrackId, Option<TrackKind>>,
 }
 
+/// Meta informationa about the video samples.
+#[derive(Clone, Debug)]
+pub struct SampleStatistics {
```

Comment:
This to me sounds like the statistics of a single sample

Reply:
`SamplesStatistics`? Dunno what to call it